src/dpr/components/bookmark-toggle/style.scss

```diff
@@ -0,0 +1,112 @@
+.bookmark {
```

Comment:
Could you change this to `dpr-bookmark`, please? Just to avoid potential collision in the consumer projects.